src/objects/chart/chart.data.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const ChartData = {
+  STRING: 'string',
+  NUMBER: 0,
+  FLOAT: 0.00
+};
+
+export default ChartData;
```

Comment:
I opted to put sample data here. The Chart object imports this and exposes it to the Vue demo app.